src/scripts/create_release.sh

```diff
@@ -36,7 +36,9 @@ function create_release() {
         INSTANA_RELEASE_SCOPE='{}'
     fi
 
-    echo "${INSTANA_RELEASE_SCOPE}" > scope.json
+    JSON_STRING_FULL=$(echo $INSTANA_RELEASE_SCOPE | jq -c | jq -R)
+    INTERPOLATED_JSON=$(eval echo $JSON_STRING_FULL)
```

Comment:
**question**: I wonder why we cannot use `envsubst` as we do for the `INSTANA_RELEASE_NAME`?
We should not use `eval` here.

Reply:
Super, I don't know this command. changing it! Thanks!

Reply:
@timoinstana changed. could you please redo de review?